webpack.config.js

```diff
@@ -1,9 +1,57 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const path = require('path');
+
+const extractCSS = new ExtractTextPlugin('[name].css');
+const extractSASS = new ExtractTextPlugin('[name].css');
+
 module.exports = {
-  context: __dirname + "/app",
-  entry: "./app.js",
+  context: path.join(__dirname, '/src/model'),
+  entry: { 
+    calculator: './index.js',
+    test: "./test/index.test.js"
+  },
 
   output: {
-    filename: "app.js",
-    path: __dirname + "/dist",
+    path: path.join(__dirname + '/public'),
+    filename: '[name].bundle.js',    
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: extractCSS.extract([ 'css-loader', ]) //'postcss-loader'
+        // use: [
```

Comment:
Is `postcss-loader` needed? I was experimenting on this, left this as a reminder.